grafana-plugins/topo-plugin/README.md

```diff
@@ -8,3 +9,27 @@ After the integration plugin, configure the corresponding Prometheus data source
 
 ## What does this plugin do？
 Topology call relationships are generated based on data queried in Prometheus and can be aggregated based on namespace and workload. The topology call relationship displays indicators such as Latency、calls、error rate and Volume. You can also view indicator data of nodes。
+
+## Getting started
+
+Build topo plugin with docker
+
+```shell
+docker run -it -v $PWD:/grafana node:16-bullseye /bin/bash
+# Run the following command in the container's bash shell
+cd /grafana/topo-plugin
+yarn install
+yarn build
+# Then exit the container
+exit
+```
+
+Check you get following yields
+
+```
+dist/
+yarn.lock
+coverage/
+node_modules/
+```
+
```

Comment:
I think a section called "Getting started" should involve "How to build", "How to install" and "How to use". Since you only describe the first part, I think this section should be titled "How to build".

Reply:
Agree!